src/shared/components/cancerSummary/CancerSummaryContent.tsx

```diff
@@ -174,20 +182,21 @@ export class CancerSummaryContent extends React.Component<ICancerSummaryContentP
     }
 
     private getColors(color:string) {
-        const colors:{[id:string]:string} = {
+        const alterationToColor: Record<keyof ICancerTypeAlterationPlotData, string> = {
             mutated:"#008000",
-            amplified:"#ff0000",
-            deleted:"#0000ff",
-            multiple:"#383838",
-            fusion:"#8B00C9",
-            gain:"rgb(255,182,193)",
+            amp:"#ff0000",
             homdel:"rgb(0,0,255)",
+            hetloss:"#000",
+            gain:"rgb(255,182,193)",
+            fusion:"#8B00C9",
             mrnaExpressionUp:"#FF989A",
             mrnaExpressionDown:"#529AC8",
             protExpressionUp:"#FF989A",
             protExpressionDown:"#E0FFFF",
+            multiple:"#666"
         };
-        return this.showGenomicAlt ? (colors[color] || "#000000") : '#aaaaaa';
+        // TODO: fix ts index signature issue so we don't have to cast alterationToColor as any
+        return this.showGenomicAlt ? ((alterationToColor as any)[color] || "#000000") : '#aaaaaa';
```

Comment:
if anyone knows how to do resolve this issue, i'm all ears.  basically, if you type a collection as Recorder (with keyof) or even as  { [k:key of Whatever]: string }
then ts throws an error if you do collection[someKey]

Reply:
can you do `{[keyof ICancerTypeAlterationPlotData]:string}` ?

Reply:
tried it.  same issue.  index signature must be string or number and for some reason it won't accept that object keys are just strings

Reply:
thats dumb as heck

Reply:
i'm sure there is some way to do it.